packages/gallery/src/components/item/itemView.js

```diff
@@ -118,17 +118,20 @@ class ItemView extends GalleryComponent {
         this.props.idx
       );
     }
-    this.props.actions.eventsListener(GALLERY_CONSTS.events.ITEM_FOCUSED, {
-      styleParams: this.props.styleParams,
-      itemStyle: this.props.style,
-    });
+    this.props.actions.eventsListener(
+      GALLERY_CONSTS.events.ITEM_FOCUSED,
+      this.props
+    );
   }
 
   onBlur() {
     if (this.props.styleParams.isAccessible) {
       this.props.actions.eventsListener(GALLERY_CONSTS.events.HOVER_SET, -1);
     }
-    this.props.actions.eventsListener(GALLERY_CONSTS.events.ITEM_LOST_FOCUS);
+    this.props.actions.eventsListener(
+      GALLERY_CONSTS.events.ITEM_LOST_FOCUS,
+      this.props
```

Comment:
@yakovhaiilo What is the missing prop that is needed by the event listener?

Reply:
@nirnaor it's not a missing prop, I'm not using it but our consumers may want to use "ITEM_LOST_FOCUS" event, and it's was decided, that we send all the data when we listening to an item event.